flow360/component/results/case_results.py

```diff
@@ -27,8 +27,10 @@
 )
 from ...exceptions import Flow360ValueError
 from ...log import log
-from ..flow360_params.conversions import unit_converter
+from ..flow360_params.conversions import unit_converter as unit_converter_v1
 from ..flow360_params.flow360_params import Flow360Params
+from ..simulation.conversion import unit_converter as unit_converter_v2
```

Comment:
Does this work? I think both modules (unit_converter_v2,unit_converter_v1) are setting up same-name global variables so it might be that here the v1 unit converter is silently overwritten by v2 ones.

Reply:
good point, I am not sure. But unit tests would fail I suppose...